src/plugins/presentation_util/storybook/main.ts

```diff
@@ -6,13 +6,9 @@
  * Side Public License, v 1.
  */
 
-import { Configuration } from 'webpack';
-import { defaultConfig, WebpackConfig } from '@kbn/storybook';
+import { defaultConfigWebFinal } from '@kbn/storybook';
 
 module.exports = {
-  ...defaultConfig,
+  ...defaultConfigWebFinal,
   addons: ['@storybook/addon-essentials'],
-  webpackFinal: (config: Configuration) => {
-    return WebpackConfig({ config });
-  },
```

Comment:
webpackFinal usage in storybook main.ts somehow is causing issues with newly added dependency of ts-node most likely an issue with storybook typescript setup

storybookjs/storybook#9610